packages/newsroom-signup/src/NewsroomProfile/GrantApplication.tsx

```diff
@@ -0,0 +1,235 @@
+import * as React from "react";
+import {
+  fonts,
+  colors,
+  Checkbox,
+  Modal,
+  BorderlessButton,
+  OBSectionHeader,
+  OBSectionDescription,
+} from "@joincivil/components";
+import styled from "styled-components";
+import { connect, DispatchProp } from "react-redux";
+import { setGrant, setSkip } from "../actionCreators";
+import { StateWithNewsroom } from "../reducers";
+import { LearnMoreButton } from "./LearnMoreButton";
+
+const DialogueBox = styled.div`
+  border: 1px solid rgba(43, 86, 255, 0.4);
+  border-radius: 8px;
+  padding: 22px 24px;
+`;
+
+const DialogueHeader = styled.h4`
+  font-family: ${fonts.SANS_SERIF};
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 32px;
+`;
+
+const DialogueDescription = styled.p`
+  color: ${colors.primary.CIVIL_GRAY_1};
+  font-family: ${fonts.SANS_SERIF};
+  font-size: 15px;
+  line-height: 24px;
+`;
+
+const SmallNote = styled.p`
+  color: ${colors.accent.CIVIL_GRAY_2};
+  font-family: ${fonts.SANS_SERIF};
+  font-size: 13px;
+  line-height: 20px;
+`;
+
+const CheckboxArare = styled.div`
```

Comment:
I think should be `CheckboxCovfefe`

Reply:
lol, what?

Reply:
Haha at the time I couldn't figure out what `arare` was a typo of, but on second look I see it must be `area`.